pkg/packetbrokeragent/agent.go

```diff
@@ -291,6 +291,12 @@ func New(c *component.Component, conf *Config, opts ...Option) (*Agent, error) {
 	} else {
 		a.grpc.pba = &disabledServer{}
 	}
+
+	// get Frequency Plans Store
+	getFPS := func(ctx context.Context) (frequencyPlansStore, error) {
```

Comment:
`getFrequencyPlanStore` without a comment is ok too.

Reply:
I choose a smaller name since is a function that is created and used in a small scope, but I'll agree that it's probably more readable if it is a more complete name. So I'll change it